create_news.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE news (
+    id          INTEGER        PRIMARY KEY AUTOINCREMENT,
+    [order]     VARCHAR (1000),
+    title       VARCHAR (255),
+    href        VARCHAR (500),
+    description TEXT (1000),
+    active      BOOLEAN,
+    deleted     BOOLEAN
+);
```

Comment:
Let's keep things organized to accomodate for future changes to the database. Could you:

- rename this file to include a `YYYY-MM-dd` date? i.e. `2017-01-19_create_news.sql`
- move this file to a new `migrations` directory

That way, as new DB changes come in it'll be easier for us to keep track of what has and hasn't been applied.